datapackage_pipelines_measure/processors/add_uuid.py

```diff
@@ -0,0 +1,28 @@
+import uuid
```

Comment:
I think it's good practise to have a unique id for each row. Perhaps not completely necessary now, but will be a pain to include later. Can you see an issue with it?